jcapiv2/docs/ProvidersApi.md

```diff
@@ -0,0 +1,138 @@
+# jcapiv2.ProvidersApi
```

Comment:
This ^^^ doesn't seem to follow the pattern of the others

Reply:
I believe it's the same section vs member of the section difference, just generated in a different way